apps/src/lib/kits/maker/WebSerialPortWrapper.js

```diff
@@ -54,14 +54,18 @@ export default class WebSerialPortWrapper extends EventEmitter {
             }
           }
         }
-      });
+      })
+      .catch(error => Promise.reject('Failure to open port: ' + error));
```

Comment:
Is this (and the other catch) being added to handle a specific case, or just a general improvement?

Reply:
Mostly a general improvement. When I was debugging an issue with this, I realized that not having these `.catch` attributes made it harder to debug. Added them now to benefit ourselves in the future.